app/assets/stylesheets/govuk_publishing_components/components/_attachment.scss

```diff
@@ -84,3 +84,45 @@ $thumbnail-icon-border-colour: govuk-colour("mid-grey");
   text-decoration: none;
   cursor: help;
 }
+
+@include govuk-media-query($media-type: print) {
```

Comment:
The result of this looks very neat and tidy but I think it might be overly complicated - personally I'd just hide the icon, it doesn't really do much for print. Then the CSS would be much shorter:

```CSS
@include govuk-media-query($media-type: print) {
  .gem-c-attachment__thumbnail {
    display: none;
  }

  .gem-c-attachment__details {
    padding: 0;
  }

  .gem-c-attachment__metadata,
  .gem-c-attachment__metadata .govuk-link::after {
    font-size: 12pt;
  }

  .gem-c-attachment__link.govuk-link {
    word-break: break-word;

    &::after {
      font-size: 12pt;
    }
  }
}
```

Reply:
The icons are a quick way to determine the filetype, so I favour keeping them for both screen and print. Better comprehension of printed documents is what we're aiming for, and visuals help with that. I'd favour keeping them for this reason, but perhaps we need another design opinion here?

Reply:
The file type is written underneath the title, and I think it's less useful to know information about a link on a printed page. Fine to ask design though.

Reply:
I checked with Nikin and he's happy to remove the thumbnail, so this is now updated to match your suggestion. I've also reduced stylelint block down to a single line for the single `!important`.